frontend/src/metabase/dashboard/reducers.js

```diff
@@ -314,7 +314,7 @@ const loadingDashCards = handleActions(
       next: state => ({
         ...state,
         loadingIds: state.dashcardIds,
-        loadingStatus: "running",
+        loadingStatus: state.dashcardIds.length > 0 ? "running" : "idle",
```

Comment:
Should we maybe add a unit test for this [here](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/dashboard/reducers.unit.spec.js)?